docs/setup-guide/cluster_setup.md

```diff
@@ -0,0 +1,433 @@
+#	Edge Cluster Multi-Layer Setup and Configuration
+
+## Abstract
+The purpose of this document is how to setup and configuration hierarchical edge cluster, and describe the each step to create virtual machine, setup the port number, install kubernets, GoLang, and so on. Running Cloud core and Edge core and deployed mission and task to Edge node. Improve the Edge computing. This Cloud and Edge design is derived from cloud end to edge end for Edge System Functional Description and the Setup Requirements Specification. The intended user of this program is the edge computing user. 
+1. Virtual machine setup (create cloud core  and edge core virutal machine, and setup port),
+2. Fornax installation and configuration (install all the kubernetes compnents: kubectl, kubadm, bubelet),
+3. GoLang installlation and configuration (install all GoLang component and load the Fornax source code), 
+4. Generate machine security certification, and deployed to virtual machine, 
+5. Install CRD file in cloud core. 
+6. Run cloud-core and edge-core and deployed mission and verify the mission.
+7. You should be a root user.
+
+## 1.1 Virtual Machine Setup and Configuration (We use AWS for example)
+-	Ubuntu 18.04, Create A virtual machine as an cloud-core, Create B and C virtual machine as edge-core.
```

Comment:
"Create A virtual machine as an cloud-core, Create B and C virtual machine as edge-core."

-->

Create VM A to run the cloud cluster, and create B and C machine as edge clusters.

Reply:
updated to: Create virtual machine A to run an cloud-core, Create virtual machine B and C to run edge-core.